test/python/visualization/timeline/test_layouts.py

```diff
@@ -0,0 +1,59 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for layouts of timeline drawer."""
+
+import qiskit
+from qiskit.test import QiskitTestCase
+from qiskit.visualization.timeline import layouts
+
+
+class TestBitArrange(QiskitTestCase):
+    """Tests for layout.bit_arrange."""
+
+    def setUp(self) -> None:
+        """Setup."""
+        super().setUp()
+
+        qregs = qiskit.QuantumRegister(3)
+        cregs = qiskit.ClassicalRegister(3)
+
+        self.regs = list(qregs) + list(cregs)
+
+    def test_qreg_creg_ascending(self):
+        """Test qreg_creg_ascending layout function."""
+        sorted_regs = layouts.qreg_creg_ascending(self.regs)
```

Comment:
Is this a layout, because it determines an order to show qubits in the image?

Reply:
exactly. more specifically, this is a function that doesn't generate (drawing) objects.